### PR TITLE
Update on restrictions and workings of smart baseline

### PR DESCRIPTION
**Pull request by SkylineDennisDr**
An investigation related to a smart baseline issue at telenet https://collaboration.dataminer.services/task/255874 prompted us to add some additional info on smart baselines to the docs.

Concretely, the user stored the smart baseline in a parameter of interprete $ rawtype equal to unsigned number. This caused an issue as our baseline could be a negative number sometimes.

I've also added some more info related to the inner workings of the smart baseline.

### CODE REVIEW

File: develop/schemadoc/Protocol/Protocol.Params.Param.Alarm-type.md [hunk 1]
```diff
@@ -30,8 +30,9 @@ Prior to DataMiner 10.1.9, when a parameter has a (smart) baseline value specifi
...
+> - The alarm template baseline editor will not allow you to change the monitoring type (Normal, Relative, Absolute, or Rate).
+> - When a baseline is specified in a protocol, the baseline value is stored in a separate parameter. Although you should specify a read parameter (e.g. `<Alarm type="absolute:NOMINAL_VALUE_PID,FACTOR_PID">`), make sure that read parameter has an associated write parameter. Otherwise, it will not be possible to update the baseline value stored in that parameter. Also, the parameter in which the baseline value is stored must not have any restrictions (e.g. step size, number of decimals, high/low range, the interprete->rawtype tag should not be an "unsigned" number, etc.).
```
Comment (MariekeGO) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Alarm-type.md:34]:
```suggestion
> - When a baseline is specified in a protocol, the baseline value is stored in a separate parameter. Although you should specify a read parameter (e.g. `<Alarm type="absolute:NOMINAL_VALUE_PID,FACTOR_PID">`), make sure that read parameter has an associated write parameter. Otherwise, it will not be possible to update the baseline value stored in that parameter. Also, the parameter in which the baseline value is stored must not have any restrictions (e.g. step size, number of decimals, high/low range, etc.), and the Interprete.RawType tag should not be set to [unsigned number](xref:Protocol.Params.Param.Interprete.RawType#unsigned-number).
```

File: develop/schemadoc/Protocol/Protocol.Params.Param.Alarm-type.md [hunk 2]
```diff
@@ -30,8 +30,9 @@ Prior to DataMiner 10.1.9, when a parameter has a (smart) baseline value specifi
...
+> - The smart baseline algorithm may sometimes produce unexpected results with respect to parameter ranges. For example, consider a parameter that consistently fluctuates between 0 and 100. While approximating its behavior using a polynomial, the calculated baseline value might occasionally fall below 0 or exceed 100. To address this, we recommended configuring appropriate ranges for the parameter within the protocol. The algorithm will respect these limits during baseline calculations. For instance, if a lower range of 0 is defined in the protocol, any baseline value calculated below 0 will automatically be capped at 0.
```
Comment (MariekeGO) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Alarm-type.md:35]: Could you rephrase this so it's a bit easier to understand for the average user? I think many of our users will have no idea what a "polynomial" is.

Comment (SkylineDennisDr) [reply develop/schemadoc/Protocol/Protocol.Params.Param.Alarm-type.md]: I just pushed an update, avoiding difficult terms. Let me know in case it is not clear enough!

File: user-guide/Basic_Functionality/Protocols_and_templates/Alarm_templates/Configuring_alarm_templates/Configuring_dynamic_alarm_thresholds.md [hunk 3]
```diff
@@ -41,16 +41,18 @@ Both for "absolute" and "relative" alarm thresholds, the "normal" value has to b
...
 
      > [!NOTE]
-     > If this option is selected, the DataMiner system will calculate the median value of all average trend points in the 15-minute time window equal to the current time window and apply a polynomial regression to these median values from the last x number of days.
+     >
+     > - If a range is defined in the protocol, the baseline value is capped to ensure it stays within the specified minimum and maximum limits.
+     > - The polynomial is recalculated every day around midnight.
```
Comment (MariekeGO) [inline user-guide/Basic_Functionality/Protocols_and_templates/Alarm_templates/Configuring_alarm_templates/Configuring_dynamic_alarm_thresholds.md:55]:
```suggestion
     > - The typical daily behavior of the signal is recalculated every day around midnight.
```

**Review by MariekeGO — APPROVED** [review]
